GUIDELINES.md

```diff
@@ -0,0 +1,7 @@
+## Guidelines
+
+### File Name
+Make sure the `.tweet` file name created follows the convention `dd-mm-yyyy-<tweet-type>.tweet`
+
+### File Contents
+Make sure the number of characters are under a tweet limit. You can add emojis to the text as well.
```

Comment:
we could also add the suggestion like below?

while doing the "@" mention of some one, please make sure it is correct, as the bot currently doesn't validate the tagging

Reply:
Yes. Good catch. I will add this as well.